whitelist.txt

```diff
@@ -58,6 +58,9 @@ vm
 num
 utils
 prepend
+LSB
+MSB
```

Comment:
```suggestion
```

Reply:
Makes sense. These changes would be reflecting in #248